addons/resource.language.en_gb/resources/strings.po

```diff
@@ -19630,3 +19630,14 @@ msgstr ""
 msgctxt "#39010"
 msgid "Select sort method"
 msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#40000"
+msgid "Show movie sets in tags"
+msgstr ""
+
```

Comment:
I guess all of this is very wrong?
I don't know how you handle new strings, so I just put them here and gave them a high ID number for testing